### PR TITLE
Item mimics can spawn from the vermin infestation event

### PR DESCRIPTION
**Pull request by unid15**
- 1-2 item mimics can spawn during the vermin infestation event

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1045,3 +1046,18 @@ var/list/admin_verbs_mod = list(
 	message_admins("[key_name_admin(usr)] has stopped all media.", 1)
 
 	stop_all_media()
+
+/client/proc/launch_event()
+	set name = "Launch Event"
+	set desc = "Select any event and fire it up!"
+	set category = "Fun"
+
+	if(!holder || !config)
```
Comment (PJB3005) [inline code/modules/admin/admin_verbs.dm]: You need better permissions checking than this.

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -1045,3 +1046,21 @@ var/list/admin_verbs_mod = list(
 	message_admins("[key_name_admin(usr)] has stopped all media.", 1)
 
 	stop_all_media()
+
+/client/proc/launch_event()
```
Comment (N3X15) [inline code/modules/admin/admin_verbs.dm]: I think we already have this.

Comment (PJB3005) [inline code/modules/admin/admin_verbs.dm]: Yeah i'm honestly suprised we don't if we actually don't.

Comment (unid15) [inline code/modules/admin/admin_verbs.dm]: All I found were some individual buttons in the secrets menu (like "spawn carps", "launch meteors" etc)

Comment (N3X15) [inline code/modules/admin/admin_verbs.dm]: event_manager.dm:56
```
/client/proc/forceEvent(var/type in allEvents)
	set name = "Trigger Event (Debug Only)"
	set category = "Debug"
```

**Comment by unid15** [comment]
Removed the verb, should be ready to go

**Comment by unid15** [comment]
@N3X15

**Pull request merged**